### DISCUSSION TITLE
Store Bought Solution

### DISCUSSION THREAD

**Discussion by JJ-Foust** [HACC2021/MVP#11] [Q&A] [answered]
Researching and showing them what is available right off the shelf

**Comment by JJ-Foust** [comment]
Salesforce Option. Simple and effective. Meets all requirements.

https://appexchange.salesforce.com/appxListingDetail?listingId=a0N3u00000MRoEKEA1

**Comment by JJ-Foust** [comment]
Another option

https://qtrac.com/

**Comment by JJ-Foust** [comment]
Another Option. 
Very Expensive.

https://www.qminder.com/

**Comment by JJ-Foust** [comment]
Another option. 

Looks simple but no real data on the website. Might need to email to inquire.

https://qless.com/queue-management/

**Comment by JJ-Foust** [comment]
Have to inquire about pricing.

https://queue-fair.com/pricing

**Comment by judyn808** [comment]
To ask this question to AQS, maybe we can reshape it to: Have you considered any commercial solutions and what has prevented AQS from adopting it? That can give us insight on their desired price point or essential features.

**Reply by JJ-Foust** [reply]
Excellent will also let us know how tech savvy they are too.

**Comment by JJ-Foust** [comment]
THis is is only $50/month. 

https://nextmeapp.com/pricing-industries/

Even though I know to win we need to code it ourself this could be a good one to add to our presentation as an alternative.

**Comment by JJ-Foust** [comment, marked as answer]
Store Bought solutions would solve the problem but does not meet the requirements for the challenge
